src/frontend/src/catalog/schema_catalog.rs

```diff
@@ -483,6 +483,14 @@ impl SchemaCatalog {
         self.function_by_name.get(name)?.get(args)
     }
 
+    pub fn get_functions_by_name(&self, name: &str) -> Option<Vec<&Arc<FunctionCatalog>>> {
+        let functions = self.function_by_name.get(name)?;
+        if functions.is_empty() {
+            return None;
+        }
```

Comment:
Is this option unnecessary?

Reply:
I thought it might panic when this function returns a `Some(vec![])`. So I added these lines to prevent it.